lib/model/dbConn.js

```diff
@@ -106,13 +106,7 @@ function init(host, db, port, options, callback) {
     }
 
     function connectionAttempt(url, options, callback) {
-        logger.info(
-            context,
-            'Attempting to connect to MongoDB instance with url %j and options %j. Attempt %d',
-            url,
-            options,
-            retries
-        );
+        logger.info(context, 'Attempting to connect to MongoDB instance with url %j. Attempt %d', url, retries);
```

Comment:
An entry to CHANGES_NEXT_RELEASE should be added. I'd suggest the following one:

```
- Fix: remove sensitive MongoDB connection parameters from log traces
```